### PR TITLE
Hide fields not allowed in security

### PR DESCRIPTION
**Pull request by vadimbasko**

### CODE REVIEW

File: src/app/MainStore.ts [hunk 1]
```diff
@@ -38,10 +49,12 @@ export class MainStore {
   @action
   loadPermissions() {
     const requestId = ++this.permissionsRequestCount;
-    this.cubaREST.getPermissions()
-      .then(action((perms: PermissionInfo[]) => {
+    this.cubaREST.getRoles()
```
Comment (web-devel) [inline src/app/MainStore.ts]: Will fail on current and previous versions of REST API. Should be changed once https://github.com/cuba-platform/cuba-rest-js/issues/20 is supported

File: src/ui/form/FieldPermssionContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  if (!props.mainStore) return <></>;
+
+  const {entityName, propertyName} = props;
+  const {getAttributePermission} = props.mainStore;
+
+  const perm: EntityAttrPermissionValue = getAttributePermission(entityName, propertyName);
+  const isAllowed = perm === 'MODIFY';
+  const isReadOnly = perm === 'VIEW';
+
+  if (!isAllowed && !isReadOnly) return <></>;
```
Comment (web-devel) [inline src/ui/form/FieldPermssionContainer.tsx]: `return null` can be used

**Review by web-devel — COMMENTED** [review]
I think maybe it makes sense to move all security related stuff to a separate class. WDYT ?

File: src/app/MainStore.ts (hunk 1 above)
Comment (vadimbasko) [reply src/app/MainStore.ts]: TODO added

File: src/ui/form/FieldPermssionContainer.tsx (hunk 2 above)
Comment (vadimbasko) [reply src/ui/form/FieldPermssionContainer.tsx]: fixed

**Comment by vadimbasko** [comment]
> I think maybe it makes sense to move all security related stuff to a separate class. WDYT ?
Implemented.

**Pull request merged**